modules/busco/meta.yml

```diff
@@ -0,0 +1,52 @@
+name: busco
+description: Benchmarking Universal Single Copy Orthologs
+keywords:
+  - quality control
+  - genome
+  - transcriptome
+  - proteome
+tools:
+  - busco:
+      description: BUSCO provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs selected from OrthoDB.
+      homepage: https://busco.ezlab.org/
+      documentation: ttps://busco.ezlab.org/busco_userguide.html
+      tool_dev_url: https://gitlab.com/ezlab/busco
+      doi: "10.1007/978-1-4939-9173-0_14"
+      licence: ['MIT']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: Nucleic or amino acid sequence file in FASTA format
+      pattern: "*.{fasta}"
+  - augustus_config:
+      type: directory
+      description: AUGUSTUS config directory
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - tsv:
+      type: file
+      description: Full summary table
+      pattern: "*.{tsv}"
+  - txt:
+      type: file
+      description: Short summary text
+      pattern: "*.{txt}"
```

Comment:
As far as I know BUSCO has several modes, one of which allows for several output summaries, you can add optional output such as [here](https://github.com/nf-core/mag/blob/d4f00bc2d178c1c8dc25b576c9e77e70511d5d21/modules/local/busco.nf#L27-L28) to the main.nf

Reply:
I tested with `--auto-lineage` and it already works. The output folder contains files from all the different databases:
```
$ tree output/busco
output/busco
└── test
    ├── run_bacteria_odb10
    │   ├── full_table.tsv
    │   └── short_summary.txt
    └── run_bacteroidales_odb10
        ├── full_table.tsv
        └── short_summary.txt

3 directories, 4 files
```

Is this what you were thinking or something more?

Reply:
Each of that files in your example (4 in total) should have their own output channel. That makes the output more predictable, therefore easier to use.